upgrade-notes/storefront_20250310_094506.md

```diff
@@ -0,0 +1,4 @@
+#### Price filters for limited users ([#3850](https://github.com/shopsys/shopsys/pull/3850))
+
+- when a user tries to reach the `brand`, `flag`, `category` or `search page` without the needed rights with applied price filters, it will remove filter query and continue to original page
```

Comment:
```suggestion
- when a user tries to reach the `brand`, `flag`, `category` or `search page` without the needed rights with applied price filters and sorting, it will remove filter query and continue to original page
```
?